k8s.gcr.io/images/k8s-staging-cluster-api-aws/OWNERS

```diff
@@ -2,15 +2,16 @@
 
 approvers:
 - richardcase
-- sedefsavas
-- Skarlso
 - Ankitasw
 - dlipovetsky
+- vincepri
 
 emeritus_approvers:
 - chuckha
 - detiber
 - ncdc
 - randomvariable
 - rudoi
-- vincepri
+- sedefsavas
```

Comment:
Hey, can you make this change in the `registry.k8s.io` folder instead? This folder is frozen an no longer accepting changes

Reply:
Of course, sorry about that.

Reply:
Updated to use registry.k8s.io